ext/exif/exif.c

```diff
@@ -4756,8 +4755,8 @@ PHP_FUNCTION(exif_thumbnail)
 	exif_error_docref(NULL EXIFERR_CC, &ImageInfo, E_NOTICE, "Returning thumbnail(%d)", ImageInfo.Thumbnail.size);
 #endif
 
-	ZVAL_STRINGL(return_value, ImageInfo.Thumbnail.data, ImageInfo.Thumbnail.size);
-	if (arg_c >= 3) {
+	RETVAL_STRINGL(ImageInfo.Thumbnail.data, ImageInfo.Thumbnail.size);
+	if (z_height) {
```

Comment:
Fun fact: this means that if you pass a $width argument but not the $height argument, that the $width isn't even filled in.

Reply:
Right, I hadn't even realised this.

This is pretty suboptimal :|

Reply:
Okay, made some changes so that one can just pass `$width`